ui/src/DataService.js

```diff
@@ -13,7 +13,12 @@ const getWSUrl = () => {
 let baseUrl = '/api';
 let wsUrl = `${getWSUrl()}/api`;
 
-const fetchItems = async (url) => {
+if (process.env.NODE_ENV !== 'production') {
+  baseUrl = 'http://localhost:8080/http://localhost:4000/api';
+  wsUrl = 'ws://localhost:4000/api';
+}
```

Comment:
this should be removed.

Reply:
ok, probably merge mishap, I'll take it from master.